docs/changelog.rst

```diff
@@ -12,6 +12,9 @@ Changelog for django-SHOP
 * If an anonymous customer logs in, his current cart is merged with a cart, which has previously
   been created. This has been adopted to re-use the method Product.is_in_cart()
   in and finds it's Merge the contents of the other cart into this one, afterwards delete it.
+* Moved field ``salutation`` from :class:`shop.models.customer.BaseCustomer` into the merchant
+  implementation. Check your projects if you don't use the default customer model:
+  :class:`shop.models.defaults.customer.Customer`.
 
```

Comment:
It would be good to be a bit more specific here: What exactly do I have to do if I am not using the default customer model?